common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -474,4 +474,14 @@ trait FeatureSwitches {
     exposeClientSide = true,
   )
 
+  val ValidateEmailSignupRecaptchaTokens = Switch(
+    SwitchGroup.Feature,
+    "validate-email-signup-recaptcha-tokens",
+    "Enables validation of reCAPTCHA tokens on email signup submissions",
+    owners = Seq(Owner.withEmail("newsletters.dev@guardian.co.uk")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
should we consider adding a sell by date so we don't forget to remove this?

Reply:
Possibly! I had wondered if we'd want to be able to turn this off indefinitely though? I don't think we want a situation where this flips off without us either making captcha token checking the default, or removing it entirely.

If we do add a `sellByDate`, how far in the future do you think is sensible?

Reply:
I _think_ sell by date will just cause the build to fail rather than turning the flag on or off

Reply:
yep that sounds good to me! thanks